include/triton/Conversion/TritonGPUToLLVM/Utility.h

```diff
@@ -564,20 +551,22 @@ void storeDistributedToShared(triton::gpu::MemDescType dstTy,
 SmallVector<Value>
 lowerLdStShared(Location loc, MLIRContext *ctx, LinearLayout cvt,
                 ArrayRef<Value> valsArray, // Input for store, output for load
-                Type llvmElemTy, Value smemBase, Value affineOffset,
+                Type llvmElemTy, Value smemBase,
+                std::function<Value(Value)> smemAddrAddon, Value affineOffset,
```

Comment:
Can you add explanation of what this `smemAddrAddon` is and how we expect to set it?